style.css

```diff
@@ -0,0 +1,106 @@
+body {
+  padding: 0;
+  margin: 0;
+}
+
+header {
+  position: fixed;
+  width: 100%;
+  height: auto;
+  margin: 0;
+  padding: 0;
+}
+
+nav {
+  top: 0;
+  width: 100%;
+  height: 50px;
+  display: flex;
+  justify-content: space-between;
+}
+
+nav p {
+  margin-left: 24px;
+  font-family: poppins, sans-serif;
+  font-weight: bold;
+  font-style: normal;
+  font-size: 18px;
+  color: #6070ff;
+}
+
+img {
+  margin-right: 24px;
+  width: 20px;
+  height: 20px;
+  margin-top: 20px;
+}
+
+main {
+  width: 100%;
+  height: 90vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background-image: url(headerpic.png);
```

Comment:
- [ ] Background image is not working. Please try to fix this issue.

![image](https://user-images.githubusercontent.com/53019916/125473957-9f5a7b2d-3478-43a7-9f68-db1dacd088c4.png)